cli/ops/timers.rs

```diff
@@ -56,12 +59,16 @@ impl GlobalTimer {
     let rx = rx
       .map_err(|err| panic!("Unexpected error in receiving channel {:?}", err));
 
-    futures::future::select(delay, rx).then(|_| futures::future::ok(()))
+    let fut = futures::future::select(delay, rx)
+      .then(|_| futures::future::ok(()))
+      .boxed_local();
+    self.future = Some(fut);
   }
 }
 
 pub fn init(rt: &mut deno_core::JsRuntime) {
   super::reg_json_sync(rt, "op_global_timer_stop", op_global_timer_stop);
+  super::reg_json_sync(rt, "op_global_timer_start", op_global_timer_start);
   super::reg_json_async(rt, "op_global_timer", op_global_timer);
```

Comment:
It's not clear to me what these three ops do. Can you give a description? (maybe put it in a comment?)